docker-compose.yml

```
@@ -16,6 +16,12 @@ services:
       - private
     restart: unless-stopped
 
+  redis:
+    image: redis:latest
```

As a side node, it's always better to pin an image to a specific version, ie 
```suggestion
    image: redis:7.0.9
```

This avoids bad surprises if there are breaking changes upstream :wink: